web/src/features/charts/tooltips/AreaGraphTooltipHeader.tsx

```diff
@@ -25,7 +25,7 @@ export default function AreaGraphToolTipHeader(props: AreaGraphToolTipHeaderProp
             }}
             className="rounded-sm font-bold"
           ></div>
-          <p className="text-base">{title}</p>
+          <p className="pr-4 text-base">{title}</p>
```

Comment:
It's a tricky balance here, cause this might break things a bit for a few languages. Maybe just do `pr-2` instead?
![Screenshot 2023-08-16 at 21 32 17](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/6571ae6a-cced-4502-8653-2fea5b2a7d3f)

Reply:
I'll see if I can find something that works. But in general I think we should scale the tooltip (on desktop at least) so the title and datetime always fit.

Reply:
Makes sense if possible :)